DESCRIPTION

```diff
@@ -17,9 +17,19 @@ Depends:
     latticeExtra (>= 0.6-14),
     polynom,
     reshape
-Imports: car (>= 2.0-2), Hmisc, stats, utils, Rcpp (>= 0.12.17)
-Suggests: DEoptim, mgcv, chron, testthat, mco, cmaes, ppso, sensitivity,ggplot2,rsm,parallel,foreach,doRedis,ff,dream, knitr 
+Imports: 
+    car (>= 2.0-2),
+    Hmisc,
+    stats,
+    utils,
+    Rcpp (>= 0.12.17),
+    coda,
+    iterators
```

Comment:
Ideally both coda and iterators would be suggests
coda is used for results from MCMC, e.g. from fitByDream
iterators is only used for parallelisation features

Reply:
Done!